docs/api-docs/store-management/shipping/shipper-hq.md

```diff
@@ -17,46 +17,46 @@ These field values can be set in the Control Panel by the merchant as well as us
 * [Product Metafields Endpoint](/api-reference/catalog/catalog-api/product-metafields/createproductmetafield)
 * [Variant Metafields Endpoint](https://developer.bigcommerce.com/api-reference/catalog/catalog-api/product-variants-metafields/createvariantmetafield)
 
-## ShipperHQ Object Properties
+## ShipperHQ object properties
 
 | Property | Type | Values |

```

Comment:
It may be a good idea to add a separate column denoting field access (whether users can view and edit specific fields) instead of having **required** or **read-only** called out in the Values column of the table. 
For example:
| id | number | ID of the metafield. | read-only |

Reply:
Good idea, I updated the table.